src/pages/collaboration/collaborationMain.module.scss

```
@@ -0,0 +1,121 @@
+.ideaMarketHeader {
+  display: flex;
+  padding: 30px 0px;
+  justify-content: space-between;
+  align-items: center;
+  align-self: stretch;
+  width: 800px;
+  margin: 0 auto;
+  position: relative;
+
+  &::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 800px;
+    height: 1px;
+    background: $grey-400;
+  }
+}
+
+.titleWrapper {
+  display: flex;
+  align-items: center;
+  gap: 15px;
+}
+
+.mainTitle {
+  color: $grey-800;
+  font-family: $font-family;
+  font-size: 28px;
+  font-style: normal;
+  font-weight: 500;
```

```suggestion
  @include pretendard-28-500
```

요 페이지에 글로벌 변수 적용하면 좋을 것 같습니다